sources/plugins/fm-simplex/index_fmsimplex.mld

```diff
@@ -0,0 +1,2 @@
+{1 FM simplex plugin}
+
```

Comment:
Maybe do the same explanations here about how to use it, as is done for the ABWhy3 plugin ?

Reply:
This will be add to the documentation todo list :). This PR only focus on Parsers